### PR TITLE
Django 2.0 and python 3.6 support

### PR DESCRIPTION
**Pull request by f213**

### CODE REVIEW

**Comment by f213** [comment]
Hey @audiolion!

What do you think about my PR?

File: tests/views.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from django.views.generic.edit import CreateView, UpdateView
 
 from .models import AuthoredMock, EditoredMock
-from .forms import AuthoredMockModelForm, EditoredMockModelForm
+from .forms import AuthoredModelFormMock, EditoredModelFormMock
```
Comment (audiolion) [inline tests/views.py]: Why did you  change the names here?

File: behaviors/forms.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def __init__(self, request=None, *args, **kwargs):
     def save(self, commit=True):
         obj = super(EditoredModelForm, self).save(commit=False)
 
-        if self.request is not None and self.request.user.is_authenticated():
+        if self.request is not None and self.request.user.is_authenticated:
```
Comment (audiolion) [inline behaviors/forms.py]: this now won't work for django 1.8 which is still supported until April, we need to add a `compat.py` module for this, ill add one in for ya

File: .travis.yml [hunk 3]
```diff
@@ -9,12 +9,14 @@ env:
...
-  - TOX_ENV=py34-django-19
-  - TOX_ENV=py27-django-19
   - TOX_ENV=py35-django-110
   - TOX_ENV=py34-django-110
   - TOX_ENV=py27-django-110
+  - TOX_ENV=py27-django-111
+  - TOX_ENV=py34-django-111
+  - TOX_ENV=py35-django-111
+  - TOX_ENV=py34-django-200
+  - TOX_ENV=py35-django-200
```
Comment (audiolion) [inline .travis.yml]: missing py36 in the travis tests here

**Review by audiolion — CHANGES_REQUESTED** [review]
I have some things I want to change but thanks for kicking this off. Sorry I missed this, I don't know why but I never got a notification for this.

**Review by audiolion — APPROVED** [review]

**Pull request merged**